prow/cmd/deck/static/script.js

```diff
@@ -418,25 +422,50 @@ function redraw(fz) {
     redrawOptions(fz, opts);
 
     var lastKey = '';
-    for (var i = 0, emitted = 0; i < allBuilds.length && emitted < 500; i++) {
-        var build = allBuilds[i];
-        if (!equalSelected(typeSel, build.type)) {
+    for (var i = 0, emitted = 0; i < allBuilds.items.length && emitted < 500; i++) {
+        var build = allBuilds.items[i];
+
+        // assign default values 
+        var prop = { 
+            org: "",
+            repo: "",
+            base_ref: "", 
+            base_sha:"",
+            pulls: [
+                {
+                    "number": 0,
+                    "author": "",
+                    "sha": ""
+                }
+            ]
+        }
+ 
+        if ( Object.keys(build.spec.refs).length == 0){
+            build.spec.refs = prop;
+        } 
+
+        if (!build.spec.refs.hasOwnProperty("pulls")) {
```

Comment:
is there any case that `build.spec.refs` inherit `pulls`? if not we can change the condition to `!build.spec.refs.pulls`